com.ibm.streamsx.topology/bin/spl-python-extract.py

```diff
@@ -181,34 +181,31 @@ def setupInfoXml(languageList):
             if copiedLanguagesSet == resourceLanguageSet:
                 print('Resource section of info.xml verified')
             else:
-                print(""""Message set for the "TopologySplpyResource" is incomplete or invalid.
-                Correct the resource section in info.xml file or continue at your one's peril.
-                
-                Sample info xml:
-                """)
-                print(_INFO_XML_TEMPLATE)
+                errstr = """"ERROR: Message set for the "TopologySplpyResource" is incomplete or invalid. Correct the resource section in info.xml file.
+
+                Sample info xml:\n""" + _INFO_XML_TEMPLATE
+                sys.exit(errstr)
         else:
-            print(""""Message set for the "TopologySplpyResource" is missing.
-                Correct the resource section in info.xml file.
-                
-                Sample info xml:
-                """)
-            print(_INFO_XML_TEMPLATE)
+            errstr = """"ERROR: Message set for the "TopologySplpyResource" is missing. Correct the resource section in info.xml file.
+
+                Sample info xml:\n""" + _INFO_XML_TEMPLATE
+            sys.exit(errstr)
     except FileNotFoundError as e:
-        print("File info.xml not found. Create info.xml from template")
+        print("WARNING: File info.xml not found. Create info.xml from template")
```

Comment:
Should "create" be "creating" here? As it is the utility that it creating `info.xml`, not an instruction for the user to create it?